### DISCUSSION TITLE
InvalidOutputValueMismatch

### DISCUSSION THREAD

**Discussion by programmingninjas** [sinaptik-ai/pandas-ai#1110] [Q&A]
I'm trying to create a pie plot for all the diseases in a dataset but it shows me this error:
pandasai.exceptions.InvalidOutputValueMismatch: Value type <class 'str'> must match with type plot

**Comment by programmingninjas** [comment]
I tried Changing the prompt little bit and now it's plotting multiple same charts one at a time. When i close them all i get the same error.
Code:
```python
from pandasai import SmartDataframe
from pandasai.llm import OpenAI

llm = OpenAI() # no need to pass the API key, it will be read from the environment variable
df = SmartDataframe("healthcare_dataset.csv", config={"llm": llm})

response = df.chat("Plot a pie chart for each medical condition in the dataset.")
print(response)
```

**Reply by gventuri** [reply]
Hi @programmingninjas PandasAI is currently not meant to be used for plotting multiple charts at a time. This is probably causing the error. Does it work if you only one plot at a time?

**Reply by programmingninjas** [reply]
@gventuri Nope I'm just plotting a single chart

**Comment by androj07** [comment]
Do you have a space in the path where you execute the notebook ( where pandas.ai could need to save the plot? )If so, then this validation will fail for your path
https://github.com/Sinaptik-AI/pandas-ai/blob/e011e8ffdc8a2cd88db07c4440f331540a175648/pandasai/helpers/output_validator.py#L99
